tv_apps/smart-home/style/home.css

```diff
@@ -71,8 +71,8 @@ section {
   top: -23.14rem;
   width: 100%;
   height: 14.926rem;
-  display: none;
```

Comment:
please try `visibility:hidden` to prevent screen reader to read these two buttons.